docs/onboard-vendors-to-techpass.md

```diff
@@ -0,0 +1,100 @@
+# Onboarding Vendors
+This section guides how vendors or contractors working on Gov ICT Projects can get a TechPass account and onboard in to it.
+
+## How to get a TechPass account for vendors or contractors?
+
+If you are a vendor or contractor who does not have an active TechPass account, request one from your project manager or reporting officer. Make sure you provide your official email address, handphone number and your project name in your request. Your project manager or reporting officer in turn will work with the engaging government agency to set up a TechPass account for you.
+
+?>
+  You can use this TechPass account for projects hosted by all government agencies.<br><br>
+  If you are a public officer or a user with a <i>user_name<span>@</span>gov.sg</i> email address, please refer to [Onboarding Public Officers](onboard-public-officers-using-non-se-machines).
+
+
+## Onboarding vendors
+When a TechPass account is created, you will be notified about it in an email. This notification will have a unique TechPass username for you and an initial password will be texted to your handphone.
+
+Complete the following steps:
+
+<details> <summary> Sign in using TechPass username and initial password</summary>
+
+  1. Go to the web address(url) provided by your project manager or reporting officer to sign in to TechPass.
```

Comment:
1. Go to the web address(url) provided by your project manager or reporting officer to sign in to SGTS service with your TechPass account.

Reply:
Done.  also updated line 9.